### PR TITLE
[PNUDEV-73] Add functionality to send message to Telegram via bot

### LINKED ISSUE #73: Add functionality to send message to Telegram via bot

### PR DESCRIPTION
**Pull request by Igor-Polatajko**

### CODE REVIEW

File: src/main/java/com/pnu/dev/shpaltaif/service/telegram/TelegramMessageSendServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        this.telegramBotUserService = telegramBotUserService;
+        this.telegramMessageSender = telegramMessageSender;
+    }
+
+    @Async
+    @Override
+    public void sendMessageToCategory(String messageContent, Category category) {
+
+        sendMessageToTelegramBotUsers(messageContent, pageable ->
+                telegramBotUserService.findAllByCategory(category, pageable));
```
Comment (volodymyrvynnyk) [inline src/main/java/com/pnu/dev/shpaltaif/service/telegram/TelegramMessageSendServiceImpl.java:32]: Do not understand where `pageable` is coming from

Comment (volodymyrvynnyk) [reply src/main/java/com/pnu/dev/shpaltaif/service/telegram/TelegramMessageSendServiceImpl.java:32]: Ohh, I got it, okay

**Review by volodymyrvynnyk — APPROVED** [review]
Looks great
Just one question, can messages be sent by any authenticated user?

**Comment by Igor-Polatajko** [comment]
> Looks great
> Just one question, can messages be sent by any authenticated user?

Thanks.
Messages can be sent only by the admin.

**Pull request merged**